services/src/messaging/src/checkstyle/checkstyle-suppress-known-issues.xml

```diff
@@ -531,6 +532,12 @@
     <suppress files="src/main/java/org/openkilda/messaging/payload/flow/FlowEndpointPayload.java" lines="32" checks="JavadocParagraph"/>
     <suppress files="src/main/java/org/openkilda/messaging/payload/flow/FlowEndpointPayload.java" lines="93" checks="NeedBraces"/>
     <suppress files="src/main/java/org/openkilda/messaging/payload/flow/FlowEndpointPayload.java" lines="94" checks="NeedBraces"/>
+    <suppress files="src/main/java/org/openkilda/messaging/payload/switches/PortConfigurationPayload.java" lines="20" checks="CustomImportOrder"/>
+    <suppress files="src/main/java/org/openkilda/messaging/payload/switches/PortConfigurationPayload.java" lines="21" checks="CustomImportOrder"/>
+    <suppress files="src/main/java/org/openkilda/messaging/payload/switches/PortConfigurationPayload.java" lines="22" checks="CustomImportOrder"/>
+    <suppress files="src/main/java/org/openkilda/messaging/payload/switches/PortConfigurationPayload.java" lines="23" checks="CustomImportOrder"/>
+    <suppress files="src/main/java/org/openkilda/messaging/payload/switches/PortConfigurationPayload.java" lines="25" checks="CustomImportOrder"/>
+    <suppress files="src/main/java/org/openkilda/messaging/payload/switches/PortConfigurationPayload.java" lines="26" checks="CustomImportOrder"/>
```

Comment:
Please fix warning, don't ignore them. For newly created classes we always try to fix checkstyle issues

Reply:
without "try", just "always".

Reply:
Fixed import issues and removed this code.